packages/back-end/src/api/openapi/payload-schemas/UpdateFactMetricPayload.yaml

```diff
@@ -55,6 +55,26 @@ properties:
   inverse:
     type: boolean
     description: Set to true for things like Bounce Rate, where you want the metric to decrease
+  quantileSettings:
+    description: Controls the settings for quantile metrics (mandatory if metricType is "quantile")
+    type: object
+    required:
+      - type
+      - quantile
+      - ignoreZeros
+    properties:
+      type:
+        type: string
+        enum:
+          - "event"
+          - "unit"
+        description: Whether the quantile is over unit aggregations or raw event values
+      ignoreZeros:
+        type: boolean
+        description: If true, zero values will be ignored when calculating the quantile
+      quantile:
+        type: number
+        description: The quantile value (from 0.001 to 0.999)
```

Comment:
You can add some more validation to the schema here if you want.

```yaml
multipleOf: 0.001
minimum: 0.001
maximum: 0.999
```

For reference: https://json-schema.org/understanding-json-schema/reference/numeric